ipaserver/install/ldapupdate.py

```diff
@@ -525,19 +532,21 @@ def create_index_task(self, attribute):
         # cn_uuid.time is in nanoseconds, but other users of LDAPUpdate expect
         # seconds in 'TIME' so scale the value down
         self.sub_dict['TIME'] = int(cn_uuid.time/1e9)
-        cn = "indextask_%s_%s_%s" % (attribute, cn_uuid.time, cn_uuid.clock_seq)
+        cn = "indextask_%s_%s" % (cn_uuid.time, cn_uuid.clock_seq)
         dn = DN(('cn', cn), ('cn', 'index'), ('cn', 'tasks'), ('cn', 'config'))
 
         e = self.conn.make_entry(
             dn,
             objectClass=['top', 'extensibleObject'],
             cn=[cn],
             nsInstance=['userRoot'],
-            nsIndexAttribute=[attribute],
+            nsIndexAttribute=list(attributes),
         )
 
-        logger.debug("Creating task to index attribute: %s", attribute)
-        logger.debug("Task id: %s", dn)
+        logger.info(
```

Comment:
IMHO this should remain at debug level because it clutters up the installer-visible output:

  [7/10]: upgrading server
Creating task cn=indextask_137634898455854420_9549,cn=index,cn=tasks,cn=config to index attributes: ipaAnchorUUID, ipaOriginalUid
Indexing finished
Creating task cn=indextask_137634898518146100_9549,cn=index,cn=tasks,cn=config to index attributes: ipaallowedtarget, ipaassignedidview, ipatokenradiusconfiglink, member, memberuid, owner, seeAlso, uniquemember
Indexing finished